src/axl_io.c

```diff
@@ -558,6 +590,21 @@ int axl_file_copy(const char* src_file, const char* dst_file, unsigned long buf_
         *crc = crc32(0L, Z_NULL, 0);
     }
 
+    /* Resume the transfer to our destination file where we left off */
+    if (resume) {
```

Comment:
I don't think the crc32 value won't be computed correctly when resuming a file.  I don't think we're using crcs in AXL, so one option is to consider dropping it?  I guess the question is whether we think users would like AXL to compute crc's as an option.

Reply:
If users would like AXL to compute crcs, we'd want to keep it, but then code needs to be added to handle resume operations.

Reply:
If we're not using the CRC code, my vote would be to remove it in a separate PR and add it back in later if needed.

Reply:
Yeah, I agree we can remove it and add it back later.  Let's take those lines out in that case.

Reply:
The crc code is still here.

Reply:
Oh, I see you're suggesting to remove it in a later PR.  Sure, we can do that.